src/pages/Flags/Flags.tsx

```diff
@@ -48,7 +57,10 @@ export const Flags = ({ route }: FlagsPageProps) => {
           </Stack>
         </Card.Body>
       </Card>
-      <Button onClick={handleReset}>Reset App</Button>
+      <HStack my={4}>
+        <Button onClick={handleReset}>Reset App</Button>
+        <Button onClick={handleResetPrefs}>Reset Flags to Default</Button>
```

Comment:
```suggestion
        <Button onClick={handleReset}>Apply</Button>
        <Button onClick={handleResetPrefs}>Reset Flags to Default</Button>
```

Reply:
So the flag TECHNICALLY applies as soon as you change it, but for a user there's no need to understand the difference.